cmake/FindFuthark.cmake

```diff
@@ -0,0 +1,125 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2021-2022 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# The C language is not enabled in our project by default, but we need it to
+# compile Futhark's code.
+enable_language(C)
+
+# Try to find our dependencies, which are of course Futhark itself (required)
+# and the CUDA toolkit for compiling CUDA projects. Also notify the user of
+# what we find.
+find_program(Futhark futhark REQUIRED)
+find_package(CUDAToolkit)
+
+message(STATUS "Found Futhark compiler at ${Futhark}")
```

Comment:
Let's remove this line from here, and put the following at the end of the file instead:

```cmake
include(FindPackageHandleStandardArgs)
find_package_handle_standard_args(Futhark
   REQUIRED_VARS Futhark_EXECUTABLE)
```

This will give the message a nice CMake-y look. And very importantly, it will automatically make the module do the right thing for the `QUIET` and `REQUIRED` arguments.